doc/data_format.rst

```diff
@@ -596,6 +596,47 @@ Version history
 - 5 (20141010) Added more items.
 
 
+Fixed gentle and thrill rides
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Gentle and thrill rides consisting of a single building. FreeRCT can read block version 1.
```

Comment:
What does `Fixed` mean? Line 601 speaks about "single building", but doesn't connect that to "fixed".

If you introduce a new word with a specific meaning, explain it so everybody has the same understanding what the word means. It avoids situations where several people talk about 'fixed' but have different notions of what it means.

Currently, one can read 'fixed' as 'not broken', as 'lack of animation', as 'not changing in size' or as 'single building' (which line 601 introduces without saying whether or not that is the meaning of 'fixed').


One of the other options is of course to remove the word 'Fixed', which likely solves the problem as well.